packages/squiggle-lang/src/rescript/Utility/E.res

```diff
@@ -289,6 +289,13 @@ module R = {
     | Ok(r) => r->Ok
     | Error(x) => x->f->Error
     }
+
+  //I'm not sure what to call this.
+  let unify = (a: result<'a, 'b>, c: 'b => 'a): 'a =>
```

Comment:
I wrote this for some code, then removed the corresponding code. We could remove now, I don't care too much.